ang/test/civicase/shared/directives/panel-query.directive.spec.js

```diff
@@ -526,6 +526,23 @@
           expect(crmApi.calls.count()).toBe(0);
         });
       });
+
+      describe('when a watcher is triggered while already loading but force reload is necessary', function () {
+        beforeEach(function () {
+          panelQueryScope.loading.full = true;
+          panelQueryScope.pagination.page = 2;
+          panelQueryScope.config.forceReload = true;
+          $scope.$digest();
+        });
+
+        it('does not make an additional api call', function () {
```

Comment:
Why do we have multiple `it` if both resolves to same condition. Can't we just have a single it
```
it('does not make an additional api call', function () {
  expect(crmApi.calls.count()).toBe(1);
  expect(panelQueryScope.config.forceReload).toBe(false);
});
```

Reply:
Oops, the description was wrong. Fixed it now. It still needs to be two separate `it` statements.